backend/src/main/scala/cromwell/backend/wdl/OutputEvaluator.scala

```diff
@@ -18,22 +19,22 @@ import wom.expression.{IoFunctionSet, WomExpression}
 
 import scala.util.{Failure, Success, Try}
 object OutputEvaluator {
+  sealed trait EvaluatedJobOutputs
```

Comment:
Let's tech talk this on Tuesday please.   My concern is that we're re-implementing the `Either` and `Try` effects.